modules/bday-interactive.js

```diff
@@ -27,6 +27,9 @@ module.exports.bdayInteractive = ({ type, values }) => {
           validData(key, input.value)
           acc[user_index] = { ...acc[user_index], url: input.value }
         }
+        if (key.includes('date')) {
+          acc[user_index] = { ...acc[user_index], date: input.value }
+        }
```

Comment:
future refactor could be:

handle the `else` exception first
```
} else {
    acc[user_index] = { id: input.selected_user }
}	      
```

then
`acc[user_index] = { ...acc[user_index], [key]: input.value }`